nucypher/network/nodes.py

```diff
@@ -352,16 +352,16 @@ def read_nodes_from_storage(self) -> List:
         for node in stored_nodes:
             try:  # Workaround until #2356 is fixed
                 node_domain = node.domain.decode('utf-8')
-            except:
-                node_domain = node.serving_domain
-            if node_domain != self.learning_domain:
+            except:  # TODO: The heck is happening here?
+                node_domain = node.domain
+            if node_domain != self.domain:
```

Comment:
Does this PR fixes #2356? It seems so. If that's the case, then the try/except can be removed, as it was only a workaround until we had a unified attribute for domains.

Reply:
Agreed! 👍

Reply:
Oh uhh, right!  I almost forgot about that convo.  Yep, it fixes it.

So I'll nix this block.